interface-definitions/service_ipoe-server.xml.in

```diff
@@ -131,6 +131,30 @@
                 </properties>
                 <defaultValue>shared</defaultValue>
               </leafNode>
+              <leafNode name="start-session">
+                <properties>
+                  <help>Start session options</help>
+                  <completionHelp>
+                    <list>dhcpv4 unclassified-packet</list>
```

Comment:
Can we use dhcp on the CLI please and transcode it internally as it's done for unclassified-packet? We always use `dhcp` on the CLI for DHCPv4 related stuff. This would keep the CLI consistent.

Reply:
Done!